### DISCUSSION TITLE
Python remake for Android Devices (and also PC)

### DISCUSSION THREAD

**Discussion by TermuxMonet** [victornpb/undiscord#334] [3. Ideas]
Hi everyone! since this project has been inactive and it doesn't support Android Devices, [i've recreated it](https://github.com/HelpyFazbear/deleteDiscordMessages.py) on Python, so that anyone can run the script on any Python Interpreter (including Python Interpreters for Android). Till now, i've implemented most of Javascript Undiscord features inside the Python script.

**Comment by victornpb** [comment]
Great work!

I haven't take a deep look at the code, but I’m interested in seeing this how you implemented a few things in python.

I love the ASCII logo btw!
